src/rng.js

```diff
@@ -1,7 +1,13 @@
 import crypto from 'crypto';
 
-const rnds8 = new Uint8Array(16);
+const poolSize = 16;
+const rnds8Pool = new Uint8Array(16 * poolSize);
```

Comment:
With apologies for being somewhat mulish here, `poolSize` just isn't adding any value here.  The main reasons for declaring a constant are ...

* Centralize a commonly used property
* Place a configurable property at the top of file where it's easy to find
* Provide a descriptive name to a configurable property

`poolSize` isn't used elsewhere, is only used at the top of file, and is in units that are somewhat confusing. ("Actual pool size is 16X poolSize, wut?!?")

(Also, if you're going to insist on keeping it, rename to `POOL_SIZE`)

```suggestion
const rnds8Pool = new Uint8Array(256); // # of random values to pre-allocate
```

Reply:
My intention was to clearly stress number of pre-allocated chunks of random numbers present in the pool. But I'm fine with the proposed change. Applied it in 742b9a8fb5752808be1bb764bee8e0908aa00a15

Thanks for the proposed improvements.